pages/instructorQuestionManualGrading/instructorQuestionManualGradingNextInstanceQuestion.js

```diff
@@ -3,24 +3,24 @@ const express = require('express');
 const router = express.Router();
 const path = require('path');
 const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
-const {sqlDb, sqlLoader} = require('@prairielearn/prairielib');
-const sql = sqlLoader.loadSqlEquiv(__filename);
+const {sqlDb} = require('@prairielearn/prairielib');
 
 router.get('/', (req, res, next) => {
-    const params = {
-        assessment_question_id: res.locals.assessment_question_id,
-        assessment_id: res.locals.assessment.id,
-    };
+    const params = [
+        res.locals.assessment.id,
+        res.locals.assessment_question_id,
+        res.locals.authn_user.user_id,
+    ];
     // Unmarked instance question df. Is last created submission of instance question AND has null graded_at value.
-    sqlDb.queryZeroOrOneRow(sql.get_next_unmarked_instance_question, params, (err, result) => {
+    sqlDb.callZeroOrOneRow('instance_questions_update_submission_for_manual_grading', params, (err, result) => {
```

Comment:
Below we assume that `rows[0]` exists. We should use `callOneRow()` to ensure this.

Reply:
Done.

Reply:
Done.